guppylang/error.py

```diff
@@ -79,7 +28,7 @@ class GuppyTypeInferenceError(GuppyError):
     """Special Guppy exception for type inference errors."""
 
 
-class MissingModuleError(GuppyError):
+class MissingModuleError(Exception):
```

Comment:
not a GuppyError?

Reply:
Yes, `GuppyError` is now only for stuff that goes wrong during compilation. This `MissingModuleError` is thrown immediately when you try to do `guppy.load()` with something that is not a module